### ISSUE TITLE
Export HMM model and use of validation test with the HMM.

### ISSUE THREAD

**Issue by ChataingT** [BelloneLab/lisbet#14]
Hi, 
I have trained LISBET and the HMM over my training set. I want to project my validation set through the embedding and the clustering of HMM but the hmm model is not "saveable" and there is no option to pass a dataset only for a "hmm.predict()". 
Could we add a feature either to export/import the hmm model and do an "only prediction" or have the ability to give a only_testing_dataset in betman segment_motifs so it not used in "hmm.fit()" but in "hmm.predict()" ?

**Comment by gchindemi** [comment]
This is an interesting scenario.

We did not implement an inference pipeline for the HMM model, as we consider it almost a post-processing step (i.e., clustering).
Furthermore, the prototype selection uses a collection of HMM models, so the user would need to keep all of them around to perform inference at a later time.

In our view, after the user has identified one or more prototypes of interest, they would fine-tune a regular LISBET classification model using the prototype data as training labels.
In other words, we did not intend for the HMM to be used for inference as is.

That being said, I don't see any issue with allowing the user to save and reuse HMM models, as long as the CLI remains minimal and consistent, and the different approaches are properly documented.
Please feel free to create a pull request!
